src/payment/_style.scss

```diff
@@ -9,4 +9,20 @@
   label {
     font-weight: normal;
   }
+  .form-group {
+    .card-icon {
+      position: relative;
+      float: right;
+      top: -27px;
+      right: 28%;
+      margin-right: -15%;
+    }
+    .lock-icon {
```

Comment:
might be good to add `pointed-events: none` to these to prevent weird selection states.